### PR TITLE
Po avatar badge

### PR DESCRIPTION
**Pull request by shruthipatil11**
avatar and badge UI components

### CODE REVIEW

File: src/app/docs/docs.css [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+.components > p,
+.components span > span {
+  font-size: 1.05rem;
+  font-weight: 400;
+  line-height: 1.5rem;
+}
+
+.components > span span b,
+.components > p b {
+  color: #2e2e2e;
```
Comment (ranitasahaa1230) [inline src/app/docs/docs.css]: Use root variables over here

File: src/app/docs/docs.css [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+  border: 1px solid white;
+  background-color: #65ccb9;
```
Comment (ranitasahaa1230) [inline src/app/docs/docs.css]: Use root variables over here

File: src/app/docs/docs.html [hunk 3]
```diff
@@ -0,0 +1,255 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (ranitasahaa1230) [inline src/app/docs/docs.html:6]: You can add description in your meta tag, Like author desc, page desc, details

File: src/app/docs/docs.css [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+  --DARK-GRAY: #9e9e9e2b;
+  --DISABLED-GREY: #565454;
+
+  --STEEL-GREY: #41464b;
+
+  --ZIRCON-GREY: #e2e3e5;
+
+  --ZUMTHOR-GREY: #d3d6d8;
+
+  --FUN-GREEN-SUCCESS: #0f5132;
```
Comment (ranitasahaa1230) [inline src/app/docs/docs.css]: Why have you written all these variables in block letters? Any reason?  Also, its best to name the CSS variables in a generic manner. We should not add the value (example gren, white, grey) in the variable name.

**Review by ranitasahaa1230 — COMMENTED** [review]
Do work on these suggestions

File: src/app/components/avatar/avatar.css [hunk 5]
```diff
@@ -0,0 +1,29 @@
+.avatar img {
+  border-radius: 50%;
+  /* margin-left: 0.5rem; */
+}
+
+.avatar--xl img {
+  height: 9rem;
+  width: 9rem;
+}
```
Comment (pankaj-prog) [inline src/app/components/avatar/avatar.css]: in this case giving height and width directly to the image will stretch the image if the provided img resolution is not square-shaped.

**Review by pankaj-prog — CHANGES_REQUESTED** [review]

File: src/app/docs/docs.css (hunk 1 above)
Comment (shruthipatil11) [reply src/app/docs/docs.css]: Done Ranita

File: src/app/docs/docs.css (hunk 2 above)
Comment (shruthipatil11) [reply src/app/docs/docs.css]: Done Ranita

File: src/app/docs/docs.html (hunk 3 above)
Comment (shruthipatil11) [reply src/app/docs/docs.html:6]: Added author and description meta data.

File: src/app/docs/docs.css (hunk 4 above)
Comment (shruthipatil11) [reply src/app/docs/docs.css]: I have renamed css variable names as per where the color was been applied like SUCCESS-MESSAGE .SUCCESS-MESSAGE color was applied on success messages. 
Is it not good convention to use block letters in css variable.

File: src/app/components/avatar/avatar.css (hunk 5 above)
Comment (shruthipatil11) [reply src/app/components/avatar/avatar.css]: Done pankaj.

File: src/app/docs/docs.css (hunk 4 above)
Comment (PuneetP16) [reply src/app/docs/docs.css]: It is good to use all caps, it increases readability as it distinguishes values from variables.

**Comment by PuneetP16** [comment]
image
The color of badge is not recognizable.

**Review by ranitasahaa1230 — APPROVED** [review]
Looks good to merge

**Comment by shruthipatil11** [comment]
> image The color of badge is not recognizable.

Resolved it Puneet.

**Review by PuneetP16 — APPROVED** [review]
Looks good to merge.

File: src/app/docs/docs.css (hunk 4 above)
Comment (shruthipatil11) [reply src/app/docs/docs.css]: okay.Thanks for input Puneet.

**Comment by shruthipatil11** [comment]
Merging avatar and badge to development branch

**Pull request merged**